yarn.lock

```diff
@@ -17482,6 +17498,11 @@ unc-path-regex@^0.1.2:
   resolved "https://registry.yarnpkg.com/unc-path-regex/-/unc-path-regex-0.1.2.tgz#e73dd3d7b0d7c5ed86fbac6b0ae7d8c6a69d50fa"
   integrity sha1-5z3T17DXxe2G+6xrCufYxqadUPo=
 
+unfetch@4.1.0:
```

Comment:
why is this needed?

Reply:
its just a dependency within amplitude's package.json